apps/api/scripts/setup-competition.ts

```diff
@@ -40,21 +40,26 @@ const colors = {
 /**
  * Parse selection input like "1,3,5-8" into an array of indices
  */
-function parseSelectionInput(input: string, maxIndex: number): number[] {
+function parseSelectionInput(input: string, teams: Team[]): Team[] {
```

Comment:
Updated this function to just return the selected Teams directly instead of an array of indexes later used to make indexed access into the teams array.